tests/TestListener.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Tests\Cachet;
+
+use PHPUnit_Framework_BaseTestListener;
+use PHPUnit_Framework_TestSuite;
+
+/**
+ * This is the test listener class.
+ *
+ * @author Connor S. Parks <connor@connorvg.tv>
+ */
+class TestListener extends PHPUnit_Framework_BaseTestListener
+{
+    /**
+     * A test suite ended.
+     *
+     * @param \PHPUnit_Framework_TestSuite $suite
+     *
+     * @return void
+     */
+    public function endTestSuite(PHPUnit_Framework_TestSuite $suite)
+    {
+        if ($suite->getName() !== 'Cachet Test Suite') {
+            return;
+        }
+
+        foreach (glob(__DIR__.'/../bootstrap/cache{,t}/*.php', GLOB_BRACE) as $file) {
```

Comment:
To go through the trouble of _booting another instance_? We also need to ensure tests don't mutate any other compiled or cached data like `services` and the configs in `cachet/*`.

This is all-round quicker to execute and more exhaustive.